README.md

```diff
@@ -1,9 +1,32 @@
 # VectorDB-recipes
+<br />
+Dive into building GenAI applications!
+This repository contains examples, applications, starter code, & tutorials to help you kickstart your GenAI projects.
 
-<a href="https://discord.gg/zMM32dvNtd">Discord</a> •
+- These are built using LanceDB, a free, open-source, serverless vectorDB that **requires no setup**. 
+- It **integrates into python data ecosystem** so you can simply start using these in your existing data pipelines in pandas, arrow, pydantic etc.
+- LanceDB has **native javascript library** using which you can **run vector search applications in your browser** as serverless functions!
```

Comment:
currently the TypeScript SDK cannot run inside the browser because of the Rust package. We do need to make a browser-enabled TypeScript SDK that just connects to LanceDB Cloud at sometime, but it's just not there now.

Reply:
ohh okayyy😮

Reply:
Now I understand why I wasn't able to bundle the multi-modal search example as static files 🙃